portfolio/src/main/webapp/style.css

```diff
@@ -14,101 +16,310 @@
     margin-right: auto;
     width: 50%;
 }
-body {
-    background-image: url('https://img.freepik.com/free-photo/white-concrete-wall_53876-92803.jpg');
-}
-h1 {text-align: center;}
+
+/* New portfolio styling */
+
 *{
-padding:0;
-margin:0;
-text-decoration: none;
-list-style: none;
-box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+	font-family: 'Josefin Sans', sans-serif;
+	box-sizing: border-box;
 }
-
-body{
-    font-family: montserrat;
+.hero{
+	height: 100vh;
+	width: 100%;
+	background-image: url(/images/background.jpg);
+	background-size: cover;
+	background-position: center;
 }
 nav{
-    background: #0082e6;
-    height: 80px;
-    width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding-top: 45px;
+	padding-left: 8%;
+	padding-right: 8%;
 }
-label.logo{
-    color: white;
-    font-size: 35px;
-    line-height: 80px;
-    padding: 0 100px;
-    font-weight: bold;
+.logo{
+	color: white;
+	font-size: 35px;
+	letter-spacing: 1px;
+	cursor: pointer;
 }
-nav ul{
-    float: right;
-    margin-right: 20px;
+span{
+	color: #f9004d;
 }
 nav ul li{
```

Comment:
[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space between the last selector and the declaration block.

Reply:
[4.2.7](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation): Separate rules by new lines.